library/auth_queries.c

```diff
@@ -159,29 +159,31 @@ void ziti_auth_query_process(ziti_context ztx, void(*cb)(ziti_context, int, void
     mfa_auth_ctx->ztx = ztx;
     mfa_auth_ctx->cb = cb;
 
-    ZITI_LOG(INFO, "sending auth query mfa authentication event, authentication cannot proceed if not handled");
+    ZTX_LOG(INFO, "sending auth query mfa authentication event, authentication cannot proceed if not handled");
     ziti_auth_query_mfa_process(mfa_auth_ctx);
 }
 
 
 void ziti_mfa_enroll_post_internal_cb(void *empty, const ziti_error *err, void *ctx) {
     ziti_mfa_enroll_cb_ctx *mfa_enroll_cb_ctx = ctx;
+    ziti_context ztx = mfa_enroll_cb_ctx->ztx;
 
     if (err == NULL) {
         ziti_ctrl_get_mfa(&mfa_enroll_cb_ctx->ztx->controller, ziti_mfa_enroll_get_internal_cb, ctx);
     } else {
-        ZITI_LOG(ERROR, "error during create MFA call: %d - %s - %s", err->http_code, err->code, err->message);
+        ZTX_LOG(ERROR, "error during create MFA call: %d - %s - %s", err->http_code, err->code, err->message);
         mfa_enroll_cb_ctx->cb(mfa_enroll_cb_ctx->ztx, err->err, NULL, mfa_enroll_cb_ctx->cb_ctx);
         FREE(ctx);
     }
 }
 
 void ziti_mfa_enroll_get_internal_cb(ziti_mfa_enrollment *mfa_enrollment, const ziti_error *err, void *ctx) {
     ziti_mfa_enroll_cb_ctx *mfa_enroll_cb_ctx = ctx;
+    ziti_context ztx = mfa_enroll_cb_ctx->ztx;
```

Comment:
same comment here - is ztx used somehow?

Reply:
ZTX_LOG macro expects a scope defined ztx